edge/pkg/metamanager/metaserver/kubernetes/storage/storage.go

```diff
@@ -100,7 +105,8 @@ func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions)
 		return obj, nil
 	}()
 	// try local
-	if err != nil {
+	if err != nil && metaserverconfig.Config.AutonomyWithoutAuthorization {
+		klog.Warningf("[metaserver/reststorage] failed to get obj from cloud: %v; autonomy switch: %v", err, metaserverconfig.Config.AutonomyWithoutAuthorization)
```

Comment:
```suggestion
		klog.Warningf("[metaserver/reststorage] try to get obj locally as which failed to get from cloud")
```
Seems the error why failed to get from cloud has been printed above

Reply:
Removed.